config/site/src/_includes/success_stories.html

```diff
@@ -19,42 +19,75 @@ <h3 class="text-3xl font-semibold mb-6">
+            <p class="text-3xl font-extrabold text-blue-600 dark:text-blue-400 mt-4">
+              200k/s
+            </p>
+            <p class="text-sm text-gray-700 dark:text-gray-300">
+              documents/s ingested
             </p>
           </div>
           <!-- GraphQL API Latency Card -->
           <div class="bg-gray-100 dark:bg-gray-700 p-6 rounded-lg md:w-1/3">
-            <h4 class="text-xl font-bold mb-4">Query Latency</h4>
+            <h4 class="text-xl font-bold mb-4">Query</h4>
             <p
               class="text-3xl font-extrabold text-blue-600 dark:text-blue-400 mb-2">
-              &lt; 350ms</p>
-            <p class="text-sm text-gray-700 dark:text-gray-300">p99 latency</p>
+              &lt; 350ms
+            </p>
+            <p class="text-sm text-gray-700 dark:text-gray-300">
+              p99 latency
+            </p>
             <p
               class="text-3xl font-extrabold text-blue-600 dark:text-blue-400 mt-4">
-              &lt; 2.5s</p>
-            <p class="text-sm text-gray-700 dark:text-gray-300">p99.99 latency
+              &lt; 2.5s
+            </p>
+            <p class="text-sm text-gray-700 dark:text-gray-300">
+              p99.99 latency
+            </p>
+            <p class="text-3xl font-extrabold text-blue-600 dark:text-blue-400 mt-4">
+              500 QPS
+            </p>
+            <p class="text-sm text-gray-700 dark:text-gray-300">
+              queries/s
             </p>
           </div>
           <!-- Dataset Card -->
           <div class="bg-gray-100 dark:bg-gray-700 p-6 rounded-lg md:w-1/3">
             <h4 class="text-xl font-bold mb-4">Dataset</h4>
-            <p class="text-3xl font-extrabold text-blue-600 dark:text-blue-400 mb-2">
-              100TB</p>
-            <p class="text-sm text-gray-700 dark:text-gray-300">indexed data</p>
-            <p class="text-3xl font-extrabold text-blue-600 dark:text-blue-400 mt-4">
+            <p
+              class="text-3xl font-extrabold text-blue-600 dark:text-blue-400 mb-2">
+              100TB
+            </p>
+            <p class="text-sm text-gray-700 dark:text-gray-300">
+              indexed data
+            </p>
+            <p
+              class="text-3xl font-extrabold text-blue-600 dark:text-blue-400 mt-4">
               &gt; 100B</p>
-            <p class="text-sm text-gray-700 dark:text-gray-300">indexed documents</p>
-            <p class="text-3xl font-extrabold text-blue-600 dark:text-blue-400 mt-4">
-              &gt; 100 QPS</p>
-            <p class="text-sm text-gray-700 dark:text-gray-300">both query & ingestion</p>
+            <p class="text-sm text-gray-700 dark:text-gray-300">
+              indexed documents
+            </p>
+            <p
+              class="text-3xl font-extrabold text-blue-600 dark:text-blue-400 mt-4">
+              &gt; 99.999%
+            </p>
+            <p class="text-sm text-gray-700 dark:text-gray-300">
+              availability
```

Comment:
The 99.999% is specifically availability of our query API so it kinda feels like it fits more on the `Query` card. But then the cards are unbalanced. So I'm fine keeping it here unless you can think of a better solution.